Lib/email/_header_value_parser.py

```diff
@@ -2404,11 +2404,13 @@ def get_section(value):
                                         value))
     section.append(ValueTerminal('*', 'section-marker'))
     value = value[1:]
-    if not value or not value[0].isdigit():
+    # We don't use str.isdigit because only 0-9 are accepted, not super-script
+    # and other types of digits.
```

Comment:
```suggestion
    # We don't use str.isdigit because only ascii digits are allowed
```
Much simpler and to the point

Reply:
If "we need a comment" was about this comment, then I'm ok with this one. However, use ASCII instead of ascii.

Reply:
Love it, done!